src/core/tracking.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-underscore-dangle */
+
+import config from 'config';
+import log from 'core/logger';
+
+
+export class Tracking {
+
+  constructor({trackingId, trackingEnabled, _log = log} = {}) {
+    if (typeof window === 'undefined') {
+      return;
+    }
+    this._log = _log;
+    this.id = trackingId;
+    this.enabled = trackingEnabled && trackingId;
+    this.logPrefix = `[GA: ${trackingEnabled ? 'ON' : 'OFF'}]`;
```

Comment:
Should this instead use `this.enabled` since that's slightly different and what we're checking in other places?